packages/KButton/KButton.vue

```diff
@@ -288,6 +297,9 @@ export default {
       @include boxShadow(var(--red-500, color(red-500)), 0, 2px);
     }
   }
+  &.rounded {
+    border-radius: 100px;
```

Comment:
Maybe better will be use `50%` instead of strict px value?

Reply:
This is a different behaviour. We want the element to have rounded edges and using 50% will make it elliptical.